users/tests/test_user.py

```diff
@@ -72,3 +72,11 @@ def test_no_authenticated_client(client, django_user_model):
     response = client.get('/profile/')
     assert response.status_code == 302
     assert response.url == '/login/?next=/profile/'
+
+
```

Comment:
Consider adding form test, 
Consider combining this with the following?
https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Testing#forms

Reply:
I added the form tests

Reply:
Please refer this PR 
https://github.com/beyond-io/roo.me/blob/main/apartments/tests.py

I would expect using django-pytest